osu.Game/PerformFromMenuRunner.cs

```diff
@@ -165,7 +165,9 @@ private bool checkForDialog(IScreen current)
 
                 // the last dialog encountered has been dismissed but the screen has not changed, abort.
                 Cancel();
+#if DEBUG
                 notifications.Post(new SimpleNotification { Text = @"An action was interrupted due to a dialog being displayed." });
+#endif
```

Comment:
We usually use `DebugUtils.IsDebugBuild` rather than a preprocessor definition. Also, you might as well convert this to a `Logger.Log` call with "debug" log level instead of a notification.